lib/transport_web/router.ex

```diff
@@ -137,4 +151,20 @@ defmodule TransportWeb.Router do
         conn
     end
   end
+
+  defp transport_data_gouv_member(conn, _) do
+    conn.assigns[:current_user]
+    |> Map.get("organizations", [])
+    |> Enum.filter(fn org -> org["slug"] == "equipe-transport-data-gouv-fr" end)
+    |> case do
+      [] ->
```

Comment:
I think it will be better to match against a list with a map in it, so we're sure we won't match a nil

Reply:
It'll be great to have tests too.